src/ctim/generators/events.clj

```diff
@@ -0,0 +1,25 @@
+(ns ctim.generators.events
```

Comment:
In the case of CTIM, yes, because it is a lib.  The other popular generators libs follow the same pattern.